src/pymor/models/iosys.py

```diff
@@ -940,6 +940,483 @@ def get_ast_spectrum(self, ast_pole_data=None, mu=None):
             return ast_lev, ast_ews[idx], ast_rev
 
 
+class PHLTIModel(Model):
+    r"""Class for (continuous) port-hamiltonian linear time-invariant systems.
+
+    This class describes input-state-output systems given by
+
+    .. math::
+        E(\mu) \dot{x}(t, \mu) & = (J(\mu) - R(\mu)) x(t, \mu) + (G(\mu) - P(\mu)) u(t), \\
+                     y(t, \mu) & = (G(\mu) + P(\mu))^T x(t, \mu) + (S(\mu) - N(\mu)) u(t),
+
+    with :math:`E\succeq 0`, :math:`J=-J^T`, :math:`N=-N^T` and
+
+    .. math::
+        \mathcal{R} = \begin{bmatrix} R & P \\ P^T & S \end{bmatrix} \succeq 0.
+
+    All methods related to the transfer function
+    (e.g., frequency response calculation and Bode plots)
+    are attached to the `transfer_function` attribute.
+
+    Parameters
+    ----------
+    J
+        The |Operator| J.
+    R
+        The |Operator| R.
+    G
+        The |Operator| G.
+    P
+        The |Operator| P.
```

Comment:
I often see pH systems where `P` is zero. Is it ok to make it zero by default?

Reply:
Sure

Reply:
Updating this docstring would be good.